Werehouse/Werehouse.py

```diff
@@ -0,0 +1,50 @@
+helmet = 0
+rifle = 0
+medkit = 0
+
+print('Hello in our werehouse')
+while True:
+    print("We have: " + str(helmet) + " helmets, " + str(rifle) + " rifles, and " + str(medkit) + " medkits"  )
+    Temp = input("Type 1 to add or 2 for out item: ")

```

Comment:
we should never name variable `temp` - variable name should always say what is the purpose. In such case we can name it e.g. `user_choice`